example/src/Utilities/Styles.js

```diff
@@ -0,0 +1,36 @@
+import { StyleSheet } from 'react-native';
+
+const Styles = StyleSheet.create({
+  page: {
+    flex: 1,
+  },
+  content: {
+    alignItems: 'center',
+    flex: 1,
```

Comment:
Are you sure `flex: 1` is what you want? This is just the shorthand for flex-grow/flex-shrink/flex-basis https://developer.mozilla.org/en-US/docs/Web/CSS/flex

To enable "flex" features use `display: flex` https://developer.mozilla.org/en-US/docs/Web/CSS/display

Reply:
All the React Native container elements are, by default, Flex containers. There’s no need to add a display: ‘flex’.

In this case `flex: 1` is for "page" to take all available space so child item "content" item can position in a center of the screen.